dev/SplitButton/SplitButton.xaml

```diff
@@ -160,9 +162,12 @@
                                     <VisualState.Setters>
                                         <Setter Target="PrimaryBackgroundGrid.Background" Value="{ThemeResource SplitButtonBackgroundChecked}"/>
                                         <Setter Target="SecondaryBackgroundGrid.Background" Value="{ThemeResource SplitButtonBackgroundChecked}"/>
-                                        <Setter Target="Border.BorderBrush" Value="{ThemeResource SplitButtonBorderBrushChecked}"/>
+                                        <Setter Target="RootGrid.BorderBrush" Value="{ThemeResource SplitButtonBorderBrushChecked}"/>
+                                        <Setter Target="RootGrid.BorderThickness" Value="0"/>
```

Comment:
This seems weird, why are we changing the color of a board with 0 thickness? and then setting the other button's border thickness to 1 without changing their bursh color?

Reply:
Regarding the first point, you are right; I removed the borderbrush setter. Regarding the second point, we are artificially increasing the buttons size by 1 px in all dimensions to deal with the fact that the outer border shrinked. Without this, we would have the button change in size if you would check/uncheck it.

Reply:
Why doesn't the colored button have a border anymore? It's supposed to have one, according to the design specs.

Reply:
This is indeed wrong. The initial reason I removed it was that the control does not properly render when checked if we leave the border:

![Screenshot of SplitButton in checked state rendering faulty](https://user-images.githubusercontent.com/16122379/159064165-b742c154-a15c-4053-aa65-29cd67c21496.png)

I'll have to dig into this again.